app/models.py

```diff
@@ -2061,6 +2080,27 @@ class DomainMailbox(db.Model, ModelMixin):
     )
 
 
+class HibpHibpDataClass(db.Model, ModelMixin):
+    __tablename__ = "hibp_hibpdataclass"
+
+    __table_args__ = (
+        db.UniqueConstraint(
+            "hibp_id", "hibp_dataclass_id", name="uq_hibp_hibpdataclass"
+        ),
+    )
+
+    hibp_id = db.Column(db.Integer(), db.ForeignKey("hibp.id"))
```

Comment:
can we add `index=True` here to make sure the foreign key is indexed? This table is rather small so indexing wouldn't make much difference here though.

Reply:
( you might need to re-run the `sh new_migration.sh` to generate the migration script though )